modules/serverless-kn-func-invoke-reference.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * serverless/reference/kn-func-ref.adoc
+
+:_content-type: REFERENCE
+[id="serverless-kn-func-invoke-reference_{context}"]
+= kn func invoke optional parameters
+
+You can specify optional parameters for the sent request by using the `kn func invoke` CLI command flags.
+
+* `-t`, `--target`: Specifies the target instance of the invoked function, for example, `local` or `remote`.
+* `-f`, `--format`: Specifies the format of the message, for example, `CloudEvent` or `http`.
+* `--id`: A unique string identifier for the request.
+* `-n`, `--namespace`: The namespace on the cluster.
+* `--source`: Sender name for the request.
+* `--type`: Type of the request, for example, `boson.fn`.
+* `--data`: Content for the request.
+* `--file`: Path to a local file containing data to be sent.
+* `--content-type`: The MIME content type for the request.
+* `-s`, `--save`: FIXME
+* `-p`, `--path`: Path to the project directory.
+* `-c`, `--confirm`: Prompt to interactively confirm all options.
+* `-v`, `--verbose`: Print verbose output.
+* `-h`, `--help`: Prints information of usage of `kn func invoke`.
+
+In particular, you might find it useful to specify the following parameters:
+
+Event target:: The target instance of the invoked function. Accepts the `local` value for a locally deployed function, the `remote` value for a remotely deployed function, or a URL for a function deployed at an arbitrary endpoint. Defaults to `local`.
+Event message format:: The message format, such as `http` or `cloudevent`. Defaults to the format of the template that was used when creating the function.
+Event type:: The type of event being sent. You can find information about the `type` parameter that is set for events from a certain event producer in the documentation for that event producer. For example, the API server source may set the `type` parameter of produced events as `dev.knative.apiserver.resource.update`.
+Event source:: The unique event source that produced the event. This may be a URI for the event source, for example `https://10.96.0.1/`, or the name of the event source.
+Event ID:: A random, unique ID that is created by the event producer.
+Event data:: Allows you to specify a `data` value for the event sent by the `kn func invoke` command. For example, you can specify a `--data` value such as `"Hello world!"` so that the event contains this data string. By default, no data is included in the events created by `kn func invoke`.
```

Comment:
The default data is actually "Hello World"

Reply:
This particular bit is just giving an example of what value we could suppy to `--data`, but sure, let's exclaim less here too :)

Reply:
> By default, no data is included in the events created by `kn func invoke`.

My point is that by default, the string `Hello World` is included in the `data` attribute for events created by `kn func invoke`.